### PR TITLE
Add advisories for async-graphql/juniper denial of service issues

### PR DESCRIPTION
**Pull request by djc**
See #1285.

### CODE REVIEW

File: crates/async-graphql/RUSTSEC-0000-0000.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+```toml
+[advisory]
+id = "RUSTSEC-0000-0000"
+package = "async-graphql"
+date = "2022-07-21"
+url = "https://github.com/async-graphql/async-graphql/commit/521769b80039fc8043d1c9883e3d6e5b57359072"
+categories = ["denial-of-service"]
+cvss = "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H"
+aliases = ["GHSA-xq3c-8gqm-v648"]
```
Comment (pinkforest) [inline crates/async-graphql/RUSTSEC-0000-0000.md:9]: they didn't reserve CVE for this yet ?

**Review by pinkforest — APPROVED** [review]
lgtm from me. I wonder if the async-graphql one has CVE reserved yet. :rocket:

**Comment by djc** [comment]
The juniper GHSA and async-graphql GHSA both link to the same CVE. I was under the impression that the CVE was for juniper (though I don't remember why), looking up the CVE it appeared to still be "reserved by a vendor" so I couldn't check.

**Comment by pinkforest** [comment]
Thanks a lot! :purple_heart:

**Pull request merged**